seed/challenges/01-front-end-development-certification/advanced-bonfires.json

```diff
@@ -236,28 +236,32 @@
-        "assert.deepEqual(makeFriendlyDates(['2022-09-05', '2023-09-04']), ['September 5th, 2022','September 4th'], 'message: <code>makeFriendlyDates([\"2022-09-05\", \"2023-09-04\"])</code> should return <code>[\"September 5th, 2022\",\"September 4th\"]</code>.');",
-        "assert.deepEqual(makeFriendlyDates(['2022-09-05', '2023-09-05']), ['September 5th, 2022','September 5th, 2023'], 'message: <code>makeFriendlyDates([\"2022-09-05\", \"2023-09-05\"])</code> should return <code>[\"September 5th, 2022\",\"September 5th, 2023\"]</code>.');"
+        "assert.deepEqual(makeFriendlyDates(['2017-01-02', '2017-01-05']), ['January 2nd', '5th'], 'message: <code>makeFriendlyDates([\"2017-01-02\", \"2017-01-05\"])</code> 应该返回 <code>[\"January 2nd\",\"5th\"]</code>.');",
+        "assert.deepEqual(makeFriendlyDates(['2017-02-01', '2017-03-03']), ['February 1st', 'March 3rd'], 'message: <code>makeFriendlyDates([\"2017-02-01\", \"2017-03-03\"])</code> 应该返回 <code>[\"February 1st\",\"March 3rd\"]</code>.');",
+        "assert.deepEqual(makeFriendlyDates(['2016-05-11', '2017-04-04']), ['May 11th, 2016', 'April 4th'], 'message: <code>makeFriendlyDates([\"2016-05-11\", \"2017-04-04\"])</code> 应该返回 <code>[\"May 11th, 2016\",\"April 4th\"]</code>.');",
+        "assert.deepEqual(makeFriendlyDates(['2017-07-12', '2018-06-13']), ['July 12th', 'June 13th'], 'message: <code>makeFriendlyDates([\"2017-07-12\", \"2018-06-13\"])</code> 应该返回 <code>[\"July 12th\",\"June 13th\"]</code>');",
+        "assert.deepEqual(makeFriendlyDates(['2003-08-15', '2009-09-21']), ['August 15th, 2003', 'September 21st, 2009'], 'message: <code>makeFriendlyDates([\"2003-08-15\", \"2009-09-21\"])</code> 应该返回 <code>[\"August 15th, 2003\", \"September 21st, 2009\"]</code>.');",
+        "assert.deepEqual(makeFriendlyDates(['2010-10-23', '2011-10-22']), ['October 23rd, 2010', 'October 22nd'], 'message: <code>makeFriendlyDates([\"2010-10-23\", \"2011-10-22\"])</code> 应该返回 <code>[\"October 23rd, 2010\",\"October 22nd\"]</code>.');",
+        "assert.deepEqual(makeFriendlyDates(['2008-10-31', '2009-10-31']), ['October 31st, 2008', 'October 31st, 2009'], 'message: <code>makeFriendlyDates([\"2008-10-31\", \"2009-10-31\"])</code> 应该返回 <code>[\"October 31st, 2008\",\"October 31st, 2009\"]</code>.');",
+        "assert.deepEqual(makeFriendlyDates(['2004-11-17', '2005-12-25']), ['November 17th, 2004', 'December 25th, 2005'], 'message: <code>makeFriendlyDates([\"2004-11-17\", \"2005-12-25\"])</code> 应该返回 <code>[\"November 17th, 2004\",\"December 25th, 2005\"]</code>.');",
+        "assert.deepEqual(makeFriendlyDates(['2001-12-20', '2001-12-20']), ['December 20th, 2001'], 'message: <code>makeFriendlyDates([\"2001-12-20\", \"2001-12-20\"])</code> 应该返回 <code>[\"December 20th, 2001\"]</code>.');",
+        "assert.isNotArray(makeFriendlyDates(['2002-12-20', '2001-12-20']), 'message: <code>makeFriendlyDates([\"2001-12-20\", \"2001-12-20\"])</code> 不应该返回数组');"
```

Comment:
What is this assert.isNotArray? It seems to be the root cause of npm test not passing.